README.md

```diff
@@ -9,6 +9,46 @@ Ballerina SerDes Library
 
 This library provides APIs for serializing and deserializing subtypes of Ballerina anydata type.
 
+### Proto3Schema
+
+An instance of the `serdes:Proto3Schema` class is used to serialize and deserialize ballerina values using protocol buffers.
+
+#### Create a `serdes:Proto3Schema` object
+
+```ballerina
+// Define a type which is a subtype of anydata.
+type Student record {
+    int id;
+    string name;
+    decimal fees;
+};
+
+// Create a serialization object by passing the typedesc.
+serdes:Proto3Schema schema = check new (Student);
+```
+While instantiation of this object, an underlying proto3 schema generated for the provided typedesc.
+
+#### Serialization
+
+```ballerina
+Student studentValue = {
+    id: 7894,
+    name: "Liam",
+    fees: 24999.99
+};
+
+// Serialize the record value to bytes.
+byte[] serializedValue = check schema.serialize(studentValue);
+```
+A value having the same type as the provided typedesc can be serialized by invoking the `serialize` method on the previously instantiated `serdes:Proto3Schema` object. The underlying implementation uses the previously generated proto3 schema to serialize the provided value.
+
+#### Serialization
+
+```ballerina
+Student deserializedValue = check schema.deserialize(serializedValue);
+```
```

Comment:
Let's provide a complete example for this, by creating the schema and deserializing it, considering this is a separate section.

Reply:
Shall we do it like this ?

```ballerina
type Student record {
    int id;
    string name;
    decimal fees;
};

byte[] bytes = readSerializedDataToByteArray();
serdes:Proto3Schema des = check new (Student);
Student student = check des.deserialize(bytes);
```

Reply:
Updated as above